apps/right-to-rent-check/acceptance/features/landlord-agent.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('landlord agent step')
+
+Before((
+  I,
+  landlordAgentPage
+) => {
+  I.visitPage(landlordAgentPage, steps)
+});
+
+Scenario('I select Landlord, click ‘Continue’ & go to /landlord-details step', (
+  I,
+  landlordAgentPage,
+  landlordDetailsPage
+) => {
+  landlordAgentPage.selectLandlordAndSubmit();
```

Comment:
I'd probably avoid creating separate helpers for all possible options on all pages - especially when the actual functionality being abstracted away is so basic.

At least, I'd possibly look to make something more re-usable like `selectOptionAndSubmit(landlordAgentPage.fields.landlord)` that can be passed an element as an argument, and then be used on *any* page with radio group rather than adding an extra helper per option per page, which gets pretty big pretty quickly.